### PR TITLE
Default to device when switching views

### PR DESCRIPTION
**Pull request by kflores772**
@bryanhiggins

### CODE REVIEW

File: public/pages/builder.html [hunk 1]
```diff
@@ -15,11 +15,11 @@
         <label class="f-label" for="targetType">Target Type</label>
         <div id="targetType" class="cr">
             <div class="radio">
-                <input type="radio" name="targetType" value="device" <% buildSettings.device ? print('checked') : print('')%>>
+                <input type="radio" name="targetType" value="device" checked>
```
Comment (rcruz) [inline public/pages/builder.html:18]: Are we no longer remembering if the user set device/simulator?
It seems strange to do this for Debug/Release and not for Device/Simulator

File: lib/ui/views/targetSelector.js [hunk 2]
```diff
@@ -35,6 +39,7 @@ TargetSelectorView = Backbone.View.extend({
 
             // Put the new html inside of the assigned element
             view.$el.html(template);
+            view.device = true;
```
Comment (rcruz) [inline lib/ui/views/targetSelector.js:42]: Why is this being set here? Is there a case that line 31-34 does not catch?

**Comment by rcruz** [comment]
Let's reconsider this before merging. This change removes the ability for the build page to remember the user's choice of device or simulator mode. This is strange behaviour since all of the other fields on this page will remember their previous values.

**Comment by kflores772** [comment]
Closing since #208 fixes this issue.

**Pull request closed without merging**